src/CoreFoundation/CFRunLoop.cs

```diff
@@ -209,6 +209,12 @@ protected override void Dispose (bool disposing)
 	}
 #endif
 
+#if NET
```

Comment:
Maybe also add == and != operators to DIsposableObject, that way we could remove these operators from the derived classes (and also remove these pragmas)?